<ui version="4.0" >
 <class>ExtractThresholdsPanel</class>
 <widget class="QWidget" name="ExtractThresholdsPanel" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="2" column="0" >
    <widget class="QCheckBox" name="PreserveTopology" >
     <property name="text" >
      <string>Preserve Topology</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" >
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="Delete" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="toolTip" >
        <string>Delete selected values from the current set</string>
       </property>
       <property name="text" >
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="DeleteAll" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="toolTip" >
        <string>Delete all values</string>
       </property>
       <property name="text" >
        <string>Delete All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="NewValue" >
       <property name="toolTip" >
        <string>Add a single value to the current set</string>
       </property>
       <property name="statusTip" >
        <string>Add a single value to the current set.</string>
       </property>
       <property name="text" >
        <string>New Value</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0" >
    <widget class="QTreeWidget" name="Thresholds" >
     <property name="editTriggers" >
      <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="tabKeyNavigation" >
      <bool>true</bool>
     </property>
     <property name="alternatingRowColors" >
      <bool>true</bool>
     </property>
     <property name="selectionMode" >
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="indentation" >
      <number>10</number>
     </property>
     <property name="rootIsDecorated" >
      <bool>false</bool>
     </property>
     <property name="itemsExpandable" >
      <bool>false</bool>
     </property>
     <property name="sortingEnabled" >
      <bool>false</bool>
     </property>
     <property name="allColumnsShowFocus" >
      <bool>false</bool>
     </property>
     <property name="columnCount" >
      <number>2</number>
     </property>
     <column>
      <property name="text" >
       <string>Min</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Max</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>49</width>
       <height>100</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" >
    <widget class="QComboBox" name="ScalarsArray" />
   </item>
   <item row="1" column="0" >
    <widget class="QCheckBox" name="InsideOut" >
     <property name="text" >
      <string>Inside Out</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QCheckBox" name="ContainingCells" >
     <property name="text" >
      <string>Containing Cells</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
